File: service-web/src/main/java/com/yanggy/cloud/api/TestController.java
package com.yanggy.cloud.api;

import com.yanggy.cloud.Properties;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * @author derrick.yang
 * @Date 9/19/18 15:34
 */

@RestController
public class TestController {

    @Autowired
    private Properties properties;

    @GetMapping(value = "/test")
    public Object test() {
        return properties.getTest();
    }
}

File: kothlin-service/src/main/kotlin/com/yanggy/cloud/kothlin/controller/TestController.kt
package com.yanggy.cloud.kothlin.controller;

import com.yanggy.cloud.kothlin.fegin.BooksFeigin
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RestController

/**
 *@author derrick.yang
 *@Date 24/10/2018 16:07
 */

@RestController
class TestController {


    @Autowired
    lateinit var booksFeigin: BooksFeigin;
    @PostMapping(value = "/insert")
    fun insertTest(): Any {
        return booksFeigin.getBooks()
    }
    @PostMapping(value = "/testFallback")
    fun testFallback(): String {
        return "hello, kothilin"
    }
}

